.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -46,7 +47,6 @@ body:
       options:
         - master / latest-dev
         - "4.0.0"
-        - "3.1.2"
```

Comment:
If 3.1 is the last minor of our two supported versions, shouldn't we continue to collect bugs for it? They might be critical in some cases, or point to issues that are also in 4.0